apps/gaia/src/pages/gallery.tsx

```diff
@@ -197,6 +199,12 @@ const ImagePreview: FC<{
           alt="preview"
           className="w-full object-cover"
         />
+        {observations.data?.map((obv) => (
```

Comment:
useGetObservationsByImage's type might be wrong, maybe GetChildObservationByImages should be an array?